CONRTIBUTING.md

```diff
@@ -61,6 +61,28 @@ __The following approach aims to reconcile these two competing demands:__
 --------
 
 
+### Library Standards
+
+The team has agreed to consistently use several third-party Python packages.
+This list will likely be fairly dynamic as we are always looking for things
+that will increase our productivity. We have a strong preference for tools that
+follow the [Principle of Least Astonishment](http://c2.com/cgi/wiki?PrincipleOfLeastAstonishment)
+(also on [Wikipedia](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)).
+
+The list:
+
+  - [arrow](https://pypi.python.org/pypi/arrow): Date/Time utilities
+  - [bottle](https://pypi.python.org/pypi/bottle): Web application micro-framework
+  - [cryptography](https://pypi.python.org/pypi/cryptography): cryptographic recipes and primitives
+  - [eventlet](https://pypi.python.org/pypi/eventlet): concurrent networking library
+  - [pymongo](https://pypi.python.org/pypi/pymongo): MongoDB driver
+  - [redis](https://pypi.python.org/pypi/redis): Redis client library
+  - [requests](https://pypi.python.org/pypi/requests): makes http requests virtually painless
+  - [tornado](https://pypi.python.org/pypi/tornado): concurrent networking library
```

Comment:
If we are going to include tornado in this privileged list, I will ask what it accomplishes that is not covered by the following:

https://docs.python.org/3/library/asyncio.html

or

One of the many asynchronous servers with built-in bottle support. Since bottle supports eventlet, gevent, diesel, fapws, tornado, twisted, meinheld, and bjoern servers, I would vote that we pick one and stick to it. I know of our team using eventlet and tornado, but.... lets choose one?